projects/sl-questionnaire/src/lib/range-input/range-input.component.scss

```diff
@@ -100,3 +100,10 @@ input[type=range]:focus::-ms-fill-lower {
 input[type=range]:focus::-ms-fill-upper {
   background: #74A9D8;
 }
```

Comment:
@pradeepkumarcm96  can you remove unused css if the existing input for range is removed?